### DISCUSSION TITLE
When will gitextensions get the CVE-2024-31497-fixing update of the bundled PuTTY tools?

### DISCUSSION THREAD

**Discussion by ultrags** [gitextensions/gitextensions#11701] [General] [answered]
On 2024-04-12, the critical vulnerability CVE-2024-31497 regarding PuTTY 0.68 - 0.80 has been published together with a fixed version 0.81 of the PuTTY tools.

The newest available gitextensions release still bundles version older than 0.81 PuTTY tools. When will a gitextensions release with version 0.81 (or newer) PuTTY tools be available?

**Comment by pmiossec** [comment, marked as answer]
>The newest available gitextensions release still bundles version older than 0.81 PuTTY tools.

Could you confirm that?
I think that since #9386 include in v4.0.0, GitExtensions doesn't bundle PuTTY anymore and user has to take care of updating it himself.
A quick check on the portable version and the msi seems to confirm that it is not bundled.


So maybe you have to clean putty binaries from an old version of GitExtensions installed and install a not vulnerable version yourself.

**Reply by ultrags** [reply]
Yes, as you said. The version I had was probably an old version and that's why it contained putty. I completely uninstalled the current version and installed the latest version and the problem gone. Thank you for the information.
